### PR TITLE
netstandard2.0, .editorconfig

### PR DESCRIPTION
**Pull request by FurkanKambay**
- [x] Target .NET Standard 2.0 (cleaner .csproj file, PackageReference)
- [x] Update .gitfiles
- [x] Add and apply .editorconfig

Also fixed NuGet package icon and added the repo URL.

If I have your permission, I'd also like to move the project itself into its own directory so it doesn't mix with the other files; it'll be cleaner.

And let me know what you think about EditorConfig, I think every repo should have it (with/without dotnet style preferences). [Here](https://gitlab.com/snippets/1793486) is a sample .editorconfig for C# that I created.

### CODE REVIEW

**Comment by ohhsodead** [comment]
Great work, thanks a bunch. Will review when I get home. Of course you're welcome to add the editorconfig, that would help a lot. Much appreciated.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+# Space preferences
+csharp_space_after_cast = false
+csharp_space_after_colon_in_inheritance_clause = true
+csharp_space_after_comma = true
+csharp_space_after_dot = false
+csharp_space_after_keywords_in_control_flow_statements = true
+csharp_space_after_semicolon_in_for_statement = true
+csharp_space_around_binary_operators = before_and_after
+csharp_space_around_declaration_statements = false
```
Comment (FurkanKambay) [inline .editorconfig:121]:
```suggestion
csharp_space_around_declaration_statements = ignore
```
to preserve spaces in places like
https://github.com/mostlyash/codbo4-csharp/blob/b329ba187bad60f61a2491323e611a4be808de84/Utilities.cs#L11-L17

**Comment by ohhsodead** [comment]
As for project files, I'll be refactoring the code very soon and will also tidy the files into sub folders too.

**Comment by FurkanKambay** [comment]
All right then.
Have you seen my comment on the .editorconfig file? Let me know if you want to preserve that whitespace. Otherwise, this should be ready to be merged.

**Comment by ohhsodead** [comment]
Looks good to me, nice work. Thanks, it will be merged soon.

**Review by ohhsodead — APPROVED** [review]

**Pull request merged**